cscope.py

```diff
@@ -170,10 +187,32 @@ class CscopeVisiter(sublime_plugin.TextCommand):
     def __init__(self, view):
         self.view = view
 
-    def run(self, edit):
+    def run(self, edit, **args):
         if self.view.settings().get('syntax') == CSCOPE_SYNTAX_FILE:
+            if args.get('direction'):
+                d = args.get('direction')
+                r = self.view.get_regions('results')
+                if r == []:
+                    return
+                view_height = len(r) - 1
+                s = self.view.settings().get('cscope_results_sel')
+                what = {
+                    'up': -1,
+                    'down': 1,
+                    'up2': -20,
```

Comment:
If we are going to do this, I think maybe up/down/pageup/pagedown would be better than up/down/up2/down2 as far as names go.

Reply:
Agree, changed